graylog2-web-interface/src/util/AppConfig.ts

```diff
@@ -22,6 +22,16 @@ declare global {
   const IS_CLOUD: boolean | undefined;
 }
 
+export type BrandingResource = { enabled?: boolean; feed?: string | undefined };
```

Comment:
Imo in the context of resources we should call the `feed` attribute for example `url`. 

For `news` we called the attribute `feed` because it is the link for an RSS feed which regularly provides new content. E.g. for the static `contact_us` page it does not make so much sense to talk about its url as a feed.